modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/rest/api/server/application/management/v1/ApplicationManagementFailureTest.java

```diff
@@ -223,6 +223,15 @@ public void testCreateApplicationsWithConflictingInboundKeys() throws Exception
 
     }
 
+    @Test(description = "Tests whether the allowed CORS origins are validated during application creation.")
+    public void testCreateApplicationsWithInvalidAllowedOrigins() throws Exception {
+
+        String payload = readResource("create-oauth-app-with-invalid-allowed-origins.json");
+        Response createAppResponse = getResponseOfPost(APPLICATION_MANAGEMENT_API_BASE_PATH, payload);
+
```

Comment:
Put a comment  saying 
// invalid URI https//localhost.com, the colon is missing.